### PR TITLE
Update buttons

### PR DESCRIPTION
**Pull request by raivop**

### CODE REVIEW

File: grails-app/views/_download.gsp [hunk 1]
```diff
@@ -34,11 +34,11 @@
 
             <br/>
             %{--<input type="hidden" name="sourceTypeId" id="sourceTypeId" value="${sourceId}"/>--}%
-            <input type="submit" value="Download All Records" class="actionButton btn btn-small" id="downloadSubmitButton" onclick="return downloadOccurrences()"/>
+            <input type="submit" value="Download All Records" class="actionButton erk-button" id="downloadSubmitButton" onclick="return downloadOccurrences()"/>
```
Comment (filippivanov) [inline grails-app/views/_download.gsp]: Hm, why are these inputs instead of buttons?

File: grails-app/views/admin/specieslists.gsp [hunk 2]
```diff
@@ -46,15 +46,15 @@
...
                             Upload a list
                         </a>
 
-                        <a class="btn" title="My Lists" href="${request.contextPath}/speciesList/list">
+                        <a class="erk-button" title="My Lists" href="${request.contextPath}/speciesList/list">
                             My Lists
                         </a>
 
-                        <a class="btn" title="Rematch" href="${request.contextPath}/speciesList/rematch">
+                        <a class="erk-button" title="Rematch" href="${request.contextPath}/speciesList/rematch">
```
Comment (filippivanov) [inline grails-app/views/admin/specieslists.gsp]: Also, I think we're switch to ``erk-button erk-button--light``. Or is it some special case?

File: grails-app/views/_download.gsp (hunk 1 above)
Comment (raivop) [inline grails-app/views/_download.gsp]: Honestly, no idea.

File: grails-app/views/admin/specieslists.gsp (hunk 2 above)
Comment (raivop) [inline grails-app/views/admin/specieslists.gsp]: I thought erk-button is the default and anything else is special case.

**Pull request merged**